core-kotlin-modules/core-kotlin-lang-oop-4/src/test/kotlin/ExtensionFieldsAlternativesUnitTest.kt

```diff
@@ -0,0 +1,117 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+
+class ExtensionFieldsAlternativesUnitTest {
+
+    private fun String.toTitleCase(): String = this.split(" ").joinToString(" ") { it.capitalize() }
+
+    private val String.firstChar: Char
+        get() = this.first()
+
+    @Test
+    fun `test using extension functions`() {
+        val word = "baeldung kotlin"
+        assertEquals("Baeldung Kotlin", word.toTitleCase())
+        assertEquals('b', word.firstChar)
+    }
+
+    class Car {
+        var model: String = "Toyota Supra"
+            get() { return field }
+            set(value) { field = value }
```

Comment:
Did you apply the Baeldung formatter to the code? I'd expect to see this on multiple lines not a single line